src/main/java/org/itsallcode/openfasttrace/core/xml/ContentHandlerAdapter.java

```diff
@@ -24,17 +24,16 @@
 
 import java.util.logging.Logger;
 
+import org.itsallcode.openfasttrace.core.Location;
+import org.itsallcode.openfasttrace.core.xml.event.EndElementEvent;
+import org.itsallcode.openfasttrace.core.xml.event.StartElementEvent;
 import org.xml.sax.Attributes;
 import org.xml.sax.ContentHandler;
 import org.xml.sax.Locator;
 import org.xml.sax.SAXException;
 import org.xml.sax.XMLReader;
 import org.xml.sax.helpers.DefaultHandler;
 
-import openfasttrack.core.Location;
-import openfasttrack.core.xml.event.EndElementEvent;
-import openfasttrack.core.xml.event.StartElementEvent;
-
 public class ContentHandlerAdapter extends DefaultHandler implements ContentHandlerAdapterController
 {
     private final static Logger LOG = Logger.getLogger(ContentHandlerAdapter.class.getName());
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Reorder the modifiers to comply with the Java Language Specification. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AModifiersOrderCheck)